test/model/content_test.dart

```diff
@@ -633,30 +683,16 @@ void main() {
       nodes: [TextNode('t')])])]));
 
   group('parse @-mentions', () {
-    testParseInline('plain user @-mention',
-      // "@**Greg Price**"
-      '<p><span class="user-mention" data-user-id="2187">@Greg Price</span></p>',
-      const UserMentionNode(nodes: [TextNode('@Greg Price')]));
-
-    testParseInline('silent user @-mention',
-      // "@_**Greg Price**"
-      '<p><span class="user-mention silent" data-user-id="2187">Greg Price</span></p>',
-      const UserMentionNode(nodes: [TextNode('Greg Price')]));
+    testParseExample(ContentExample.userMentionPlain);
+    testParseExample(ContentExample.userMentionSilent);
 
     testParseInline('silent user @-mention, class order reversed',
       // "@_**Greg Price**" (hypothetical server variation)
       '<p><span class="silent user-mention" data-user-id="2187">Greg Price</span></p>',
       const UserMentionNode(nodes: [TextNode('Greg Price')]));
```

Comment:
I think these classes-flipped examples are a bit easier to read if they're right next to their unflipped counterparts — that way the reader can easily see that that's the only thing different. So it'd be good to convert them at the same time.